BreanaMarie/tic-tac-toe/styles.css

```diff
@@ -0,0 +1,157 @@
+body{
+	background-image: url("http://breanamarie.com/GA/space.jpg");
+	background-position:center;
+	background-repeat: no-repeat;
+	background-size: 100% 150%;
+}
+
+h1{
+	color: white;
+}
+
+p{
+	color: white;
+	text-align: center;
+	font-size: 14px;
+}
+
+#board {
+  margin-top: 10px;
+}
+
+.box {
+  border: 1px solid #63bd00;
+  height: 150px;
+}
+
+.freeSpace{
+	width: 30%;
+	height: 100%;
+	text-align: center;
+
+}
+
+.playedSpaceX{
+	width: 30%;
+	height: 100%;
+	background-image: url("http://breanamarie.com/GA/wdi23.gif");
+	background-size: 100%;
+	background-position: center;
+	background-repeat: no-repeat;
+
+
+}
+
+.playedSpaceO{
+	width: 30%;
+	height: 100%;
+	background-image: url("http://breanamarie.com/GA/spaceJammers.gif");
+	background-size:80%;
+	background-repeat: no-repeat;
+	background-position: center;
+}
+
+.winnerSpace{
+	width: 30%;
+	height: 100%;
+	color: green;
+	opacity: .3;
+}
+
+.buttonReset{
+	width: 200px;
+	height: 50px;
+	display: block;
+	margin: auto;
+	text-align: center;
+	font-family: ;
+	font-size: 20px;
+	background-color: #ffcb39;
+}
+#reset{
+	margin-top: 20px;
+}
+
+#box1 {
+	width: 33.3%;
+	height: 100%;
+	float: left;
+	border-bottom: 4px solid #63bd00;
+	border-right: 4px solid #63bd00;
+
+}
+
+#box2 {
+	width: 33.2%;
+	height: 100%;
+	float: left;
+	border-right: 4px solid #63bd00;
+	border-left: 4px solid #63bd00;
+	border-bottom: 4px solid #63bd00;
+
+}
+
+#box3 {
```

Comment:
instead of setting the styles individually for each box, are there any styles here that are common to all boxes that could be applied to a class and used on all boxes?